piwheels/importer/__init__.py

```diff
@@ -161,6 +176,7 @@ def print_state(state):
     logging.warning('Duration: %s', state.duration)
     logging.warning('Output:   %d line(s)', len(state.output.splitlines()))
     logging.warning('Files:    %d', len(state.files))
+    logging.warning('Dependencies: %s', dependencies)
```

Comment:
Minor nitpick: given dependencies is potentially sufficiently long to warrant an external file, it may be best to output this in a similar manner to the "files" set below (i.e. a list with one line per dependency), or possibly a word-wrapped comma-separated list, e.g. textwrap.fill(dependencies, width=70))